src/api/volunteerRequest.ts

```diff
@@ -26,14 +28,51 @@ const router = Router();
  *                 items:
  *                   $ref: '#/components/schemas/volunteerRequest'
  */
-router.get('/', async (req: Request, res: Response, next: NextFunction) => {
+router.get('/volunteer', async (req: Request, res: Response, next: NextFunction) => {
   try {
     res.json(await getRelevantAndOpenVolunteerRequests());
   } catch (e) {
     next(e);
   }
 });
 
+/**
+ * @openapi
+ * paths:
+ *   /api/v1/volunteer-requests/manager:
+ *     get:
+ *       operationId: getObject
+ *       responses:
```

Comment:
add here the `security` property to the swagger definition of the route, check `volunteerRequest` for examples.
it is done to indicate that the route is protected and jwt is required to access it